lib/perl/Genome/Utility/Test.pm

```diff
@@ -312,10 +323,27 @@ sub data_dir_ok {
     return $data_dir;
 }
 
+sub get_test_dir {
```

Comment:
I would agree that a lot of our testsuite is a long way from being "unit tests"!

Would it make sense to have an `assert` in `Genome::Utility::Test` that's like `ok`, but is basically:
```perl
sub assert {
  my ($test, $msg) = @_;
  if ($test) { 
    note $msg;
  } else {
    die $msg;
  }
}
```
It could be used for things that should be true, but aren't really tests of the code?  The overall effect will be the same--the test fails--but the specific problem won't get parsed and reported in test reports as a failing testcase.